packages/siop-oid4vp/lib/__tests__/SdJwt.spec.ts

```diff
@@ -84,7 +84,7 @@ function getVCs(): OriginalVerifiableCredential[] {
   return [SD_JWT_VC]
 }
 
-describe('RP and OP interaction should', () => {
+describe.skip('RP and OP interaction should', () => {
```

Comment:
Please do not just skip tests without making a reference to a ticket

Reply:
fixed